FPIS/Views/CreateProductForm.cs

```diff
@@ -1,27 +1,34 @@
 ﻿using FPIS.Models;
-using FPIS.Views;
+using FPIS.Services;
 using MaterialSkin.Controls;
-using System.Text.RegularExpressions;
 using static MaterialSkin.MaterialSkinManager;
 
 namespace FPIS.Views
 {
     public partial class CreateProductForm : MaterialForm
     {
-        bool _isDataValid = true;
+        public bool _isDataValid = true;

```

Comment:
Just a thought please... When we start writing functions to deal with these repetitive tasks like validations, our functions would return a bool causing this variable to become obsolete. Perhaps you could make ValidateProductName return a bool instead